src/playback-watcher.js

```diff
@@ -253,7 +267,7 @@ export default class PlaybackWatcher {
    * @private
    */
   checkCurrentTime_() {
-    if (this.tech_.seeking() && this.fixesBadSeeks_()) {
+    if (this.fixesBadSeeks_()) {
```

Comment:
`fixesBadSeeks_` already returns false if `seeking` is `false` on `tech` so checking `seeking` is redundant.